airflow/providers/docker/operators/docker.py

```diff
@@ -243,30 +245,39 @@ def _run_image(self):
                 tty=self.tty,
             )
 
-            lines = self.cli.attach(container=self.container['Id'],
-                                    stdout=True,
-                                    stderr=True,
-                                    stream=True)
+            self.cli.start(self.container["Id"])
 
-            self.cli.start(self.container['Id'])
+            result = self.cli.wait(self.container["Id"])
+
+            lines = [
+                templ.decode("utf-8").strip() if hasattr(templ, "decode") else str(templ)
+                for templ in self.cli.logs(
+                    self.container["Id"], stream=True, stdout=True, stderr=False
+                )
+            ]
```

Comment:
So this will load the full logs of the application into memory, right? So if the application log file is 1 GB in size Airflow will start using 1GB extra RAM?